crates/egui/src/widgets/text_edit/builder.rs

```diff
@@ -1025,6 +1032,40 @@ fn events(
 
 // ----------------------------------------------------------------------------
 
+fn ime_enabled_filter_events(events: Vec<Event>) -> Vec<Event> {
+    let mut filter_events: Vec<Event> = Vec::new();
+
+    for event in events {
+        match event {
+            Event::Key { repeat: true, .. }
+            | Event::Key {
+                key: Key::Backspace,
+                ..
+            } => {}
+            _ => filter_events.push(event),
+        }
+    }
+
+    filter_events
```

Comment:
This could be a `.filter(…).collect(…)` or a `.retain(…)`

Reply:
> This could be a `.filter(…).collect(…)` or a `.retain(…)`

It has been modified.